src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,24 @@
+package racingcar;
+
+public class Car {
+
+    private final String name;
+    private int position;
+
+    public Car(String name) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException();
+        }
+        this.name = name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void move(final MovingStrategy moveingStrategy) {
+        if  (moveingStrategy.movable()) {
+            position++;
+        }
+    }
+}
```

Comment:
파일의 마지막 라인에 개행을 하는것은 어떨까요?


```suggestion
    }
}

```

래퍼런스 : https://jojoldu.tistory.com/673

Reply:
오 알려주셔서 감사합니다. 참고해보겠습니다 !

Reply:
수정했습니다~

Reply:
안녕하세요 준용님, 해당 미션은 머지되어서 다음 단계로 넘어가주셔도 될 것 같습니다!